### PR TITLE
Jared test2

### PR DESCRIPTION
**Pull request by mrjkey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # FairFareFinder
 
+another change here then
```
Comment (Tris20) [inline README.md:3]: Demo complete; we can delete :)
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -131,3 +133,5 @@ go run main.go
 ```
 go get gopkg.in/yaml.v2
 ```
+
+another merge conflict thing maybe
```
Comment (Tris20) [inline README.md:136]: Demo complete; we can delete

```suggestion

```

File: README.md (hunk 1 above)
Comment (mrjkey) [reply README.md:3]: nah, brah, full a shit

Comment (Tris20) [reply README.md]: excuse me sir, this is a Dee's

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # FairFareFinder
 
+
 rebasing this
```
Comment (Tris20) [inline README.md:4]: Demo complete; we can delete
```suggestion

```

**Review by Tris20 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mrjkey) [reply README.md:136]: bvoooooooo

Comment (Tris20) [reply README.md:135]: What are you? A barking cow?!

**Pull request merged**